gateway/handler_error.go

```diff
@@ -20,6 +23,54 @@ const (
 	defaultContentType    = headers.ApplicationJSON
 )
 
+const (
+	OAuthAuthorizationFieldMissing config.TykErrorType = "oauth.auth_field_missing"
+	OAuthBearerTokenMalformed      config.TykErrorType = "oauth.bearer_token_malformed"
+	OAuthKeyNotAuthorised          config.TykErrorType = "oauth.key_not_authorised"
+	OAuthClientDeleted             config.TykErrorType = "oauth.client_deleted"
+)
+
+var TykErrors = map[config.TykErrorType]config.TykError{
```

Comment:
We probably should initialize map somewhere else. And each file can define its errors inside `init` function or similar.